app/static/src/app/store/run/mutations.ts

```diff
@@ -102,6 +103,20 @@ export const mutations: MutationTree<RunState> = {
         delete state.parameterSetResults[parameterSetName];
     },
 
+    [RunMutation.SwapParameterSet](state: RunState, parameterSetName: string) {
+        const paramSet = state.parameterSets.find((set: ParameterSet) => set.name === parameterSetName);
+        if (paramSet && state.parameterValues && state.resultOde) {
+            // swap values
+            const temp = state.parameterValues;
+            state.parameterValues = paramSet.parameterValues;
+            paramSet.parameterValues = temp;
+            // swap results
+            const tempResult = state.resultOde;
+            state.resultOde = state.parameterSetResults[parameterSetName];
+            state.parameterSetResults[parameterSetName] = tempResult;
```

Comment:
could use destructuring here to make this a bit easier to read:

```
[state.parameterValues, other.parameterValues] = [other.parameterValues, state.parameterValues];
[state.resultOde, other.resultOde] = [other.resultOde, state.resultOde];
```

(with the minor change of variable name of `paramSet` to `other`). Swapping variables is always gross

Reply:
sorry, I missed that - just leave it as it is :)

Reply:
sorry, I missed that - just leave it as it is :)